$NetBSD: patch-ba,v 1.1 2007/08/26 17:26:51 heinz Exp $

 Both submitted upstream:
 - no need for a hard-coded path, the appropriate variables already exist
 - the documentation conflicts with the code

--- sa-update.raw.orig	2007-08-08 15:19:47.000000000 +0200
+++ sa-update.raw
@@ -1322,7 +1322,7 @@ Options:
                           keyring. Use multiple times for multiple files
   --updatedir path        Directory to place updates, defaults to the
                           SpamAssassin site rules directory
-                          (default: /var/lib/spamassassin/<version>)
+                          (default: @@LOCAL_STATE_DIR@@/@@VERSION@@)
   -D, --debug [area=n,...]  Print debugging messages
   -V, --version           Print version
   -h, --help              Print usage message
@@ -1441,7 +1441,7 @@ Note: use of this option automatically e
 
 By default, C<sa-update> will use the system-wide rules update directory:
 
-        @@LOCAL_STATE_DIR@@/spamassassin/@@VERSION@@
+        @@LOCAL_STATE_DIR@@/@@VERSION@@
 
 If the updates should be stored in another location, specify it here.
 
